net/WebSocketHandler.hpp

```diff
@@ -241,21 +246,52 @@ class WebSocketHandler : public ProtocolHandlerInterface
     void shutdown(const StatusCodes statusCode = StatusCodes::NORMAL_CLOSE,
                   const std::string& statusMessage = std::string(),
                   bool hardShutdown = false)
+    {
+        shutdownImpl(statusCode, statusMessage, hardShutdown, false);
+    }
+
+    /// Don't wait for the remote Websocket to handshake with us; go down fast.
+    void shutdownAfterWriting()
+    {
+        shutdownImpl(WebSocketHandler::StatusCodes::NORMAL_CLOSE, std::string(),
+                     true /* hard async shutdown & close */, false);
+    }
+
+    /// Returns true if the underlying socket is connected.
+    bool isConnected() const {
```

Comment:
Formatting is all out of shape.

Reply:
Yeah, as it was all out of shape (I just moved the methods into same-access-group).
Reminder - for now, while we don't want to break old style necessarily ... and try to be nice formatting wise,
we have no strict rule.